docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -1523,7 +1525,7 @@ testImplementation("io.quarkus:quarkus-junit5")
 ----
 
 ifndef::no-deprecated-test-resource[]
-[[integration-testing-wiremock]]
+[[code-flow-integration-testing-wiremock]]
```

Comment:
Looks like it was deprecated sometime ago, I missed that, but yes, as far as I recall, `KeycloalTestResourceLifecycleManager` is the only deprecated test option.

Reply:
Thanks, @sberyozkin. These conditionals were added in https://github.com/quarkusio/quarkus/pull/41712. I am proposing to remove them in https://github.com/quarkusio/quarkus/pull/41977. Does that address this issue?